### ISSUE TITLE
Comments still being converted from double-slash to star-slash

### ISSUE THREAD

**Issue by jhildenbiddle** [mrmlnc/vscode-autoprefixer#92]
Re-opening issue originally discussed in #91.

Unfortunately, this does not appear to be resolved.

- I have disabled all VSCode extensions except for Autoprefixer
- I have upgraded the Autoprefixer extension to 2.1.1
- I have tried using an external version of autoprefixer by installing it via NPM and setting `autoprefixer.findExternalAutoprefixer` to `true` in VSCode
- I have verified that the issue is not related to autoprefixer by comparing the output of the extension and postcss-cli + autoprefixer

SCSS:

```scss
// Comment
body {
    display: flex;
    // color: #000;
}
```

Processed using postcss-cli and autoprefixer:

```scss
// Comment
body {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    // color: #000;
}
```

VSCode with Autoprefixer plugin (2.0.0 & 2.1.1):

```scss
/* Comment*/

body {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    /* color: #000;*/
}
```

This also breaks [SASSDoc](http://sassdoc.com/) "triple-slash" comments (again, no issue using postcss-cli + autoprefixer).

Before:

```scss
/// Comment
body {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    /// color: #000;
}
```

After

```scss
/*/ Comment*/
body {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    /*/ color: #000;*/
}
```

My hunch is that the `postcss-safe-parser` plugin used by your extension is the issue, fixing what it sees as invalid CSS comments ("//" and "///") and converting them to valid CSS comments. That makes sense when compiling to CSS, but not when autoprefixing LESS, Stylus, or SASS/SCSS code in an editor.

Thanks again for taking a look at this!

**Comment by mrmlnc** [comment]
Please, don't worry. I was on vacation without access to the internet ☺️

**Comment by mrmlnc** [comment]
Well, now must be solved by 2.1.2.

**Comment by jhildenbiddle** [comment]
All set! Thanks for taking care of this, @mrmlnc !
